scripts/imageIDI.py

```diff
@@ -194,6 +197,23 @@ def main(args):
         plt.show()
         
         if args.fits is not None:
+            ## Make Zenith PC so SV can overwrite it
+            zra = lo.sidereal_time()*180/numpy.pi
+            zdec = lo.lat*180/numpy.pi
+
+            ## Create Phase Pointing in RA/DEC for LWA-SV if requested
+            if args.lwasv:
+                obstime = astropy.time.Time(jdList[0], format='jd')
+                loc = EarthLocation(lat=34.3484, lon=-106.8858, height=1477.8*u.m) #Approximates LWA-SV location
+                z_skycoord = SkyCoord(AltAz(az = 90.9743763*u.deg, alt = 88.0666283*u.deg, obstime = obstime, location = loc))
+                zra, zdec = z_skycoord.transform_to('icrs').ra, z_skycoord.transform_to('icrs').dec
+                zra = zra.deg
+                zdec = zdec.deg
+
+            
+            ### Make ImgWPlus object for proper pixel scale
+            fits_imgwp = utils.ImgWPlus(size=uv_size,res=res)
+        
```

Comment:
Yeah, we should use the source information in the FITS-IDI file for this.

Reply:
Is this preserved somewhere in the metadata or would you want me to recalculate the equivalent using what is available?

Reply:
It's definitely in the input data file and probably in the `CorrelatedData` instance.

Reply:
Yep, `VisibilityDataSet` has a `phase_center` attribute that stores what the phase center is.